swift/ZKSyncSDK/ZKSyncSDK/ZKSyncSDK+Types.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  ZksCryptoSecretKey.swift
+//  ZKSyncSDK-UI
+//
+//  Made with ❤️ by Matter Labs on 10/23/20
+//
+
+import Foundation
+
+public class ZKSecretKey {
+    private var content: Data
+    
+    init(_ content: Data) {
+        self.content = content
+    }
+    
+    convenience init(_ content: [UInt8]) {
+        self.init(Data(content))
+    }
+    
+    public func data() -> Data {
+        return content
+    }
+    
+    public func base64String() -> String {
```

Comment:
Hm. Is base64 encoding required for the some kind of `swift` needs? If encode primitive to strings, I'd expect it to be hexadecimal encoding.

Reply:
Ok, added `hexEncodedString` function